File: Java/TP4-4/src/Group.java
public class Group extends Usuario {

	public Group(String login) {
		super(login);
	}

	public void menuGroup() {
		System.out.println("Menu:");
		System.out.println("1-Permissões");
		System.out.println("2-Opções de Grupo");
		System.out.println("8-Trocar de Usuário");
		System.out.println("9-Sair");
		System.out.println("Opção: ");
	}

	@Override
	public void despedidaUsuario() {
		System.out.println("Log-off " + super.getLogin());
	}

	@Override
	public void descricaoUsuario() {
		System.out.println("Pode participar de Grupos.");
	}

	public void groupBoard() {
		System.out.println("Você remover Usuários deste Grupo");
	}
}

File: Kotlin/TPK4-4/bin/Group.kt
class Group(login: String) : Usuario(login) {
	fun menuGroup() {
		println("Menu:")
		println("1-Permissões")
		println("2-Opções de Grupo")
		println("8-Trocar de Usuário")
		println("9-Sair")
		println("Opção: ")
	}

	
	fun despedidaUsuario() {
		System.out.println("Log-off " + super.getLogin())
	}

	fun descricaoUsuario() {
		println("Pode participar de Grupos.")
	}

	fun groupBoard() {
		println("Você remover Usuários deste Grupo")
	}
}